src/basic/xattr-util.c

```diff
@@ -140,7 +140,12 @@ static int parse_crtime(le64_t le, usec_t *usec) {
 }
 
 int fd_getcrtime_at(int dirfd, const char *name, usec_t *ret, int flags) {
-        struct_statx sx;
+        struct_statx sx
+#if HAS_FEATURE_MEMORY_SANITIZER
+                = {} /* This is just for the memory sanitizer. Remove after it learns
+                      * about statx. */
```

Comment:
maye add a `#warn` here?

Reply:
What would I warn about? Running under msan is not an error.

Reply:
just that as a reminder that this is still in there, and should be removed one day. i.e.

```
#warning "Initializing struct statx explicitly zero, to work around msan limitation. Please remove as soon as msan has been updated to not need this any longer"
```